torch/_torch_docs.py

```diff
@@ -3178,6 +3178,54 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins=10, weights=None, min=0, max=0, density=False) -> Tensor
+
+Computes the histogram of a tensor.
+
+Args:
+    {input}
+    bins (int or 1D Tensor): If int, defines the number of bins. If Tensor,
+        defines a monotonically increasing sequence of bin edges, allowing
+        for non-uniform bins. Default: 10
+    min (float): lower end of the range (inclusive), ignored if bins is a Tensor
+    max (float): upper end of the range (inclusive), ignored if bins is a Tensor
+    weights (Tensor): optional, weight for each value in the input tensor.
+        Should be of same size as input tensor.
+    density (bool): If ``False``, the result will contain the number of samples in
+        each bin. If ``True``, the result is the value of the
```

Comment:
There has to be a simpler way to describe what density does. This description (which is NumPy's description) is nonsense.

Reply:
I believe the updated version might be a simpler way to describe it?

Reply:
Sorry, which updated version?

My main complaint with the NumPy documentation for this function is they don't do a good job relating the actual computation to the output. Like, what is this the pdf of? Also, they use ridiculous language like "bins of unity width." 

Isn't the computation just (num elements in bin) / (total_elements * bin_size)?

Reply:
Yes, it is. Made that change to the documentation.